DirectEmployers/MyJobs
# Fix broken Redux connection.

## What's this PR do?

Fixes a bug in the report type selection bar.

## How should this be manually tested?

Click around in the reporting menus. The screen switches should make sense. You should not see any analytics foo.

## Any background context you want to provide?

Looks like we renamed some redux state and missed this.
